pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   glob: ^2.0.1
   html: ^0.15.0
   logging: ^1.0.2
-  markdown: ^4.0.0
+  markdown: ">=4.0.0 <6.0.0"
```

Comment:
Hmm, why is it trying to widen the dep here? I thought that `increase-if-necessary` was the default. This behavior looks like what I thought `widen` would do?

cc @jonasfj; should we use `increase-if-necessary` explicitly in the config file? Or, do we actually want `increase`?

Reply:
For context, version `5.0.0` is out now, so I would have expected dependabot to put `^5.0.0` here.

Reply:
I don't think its necessary to increase. This commit of dartdoc is compatible with 4.0.0 and 5.0.0.

Reply:
@srawlins - feel free to land this as is if this is the dep change you'd like :) I'd like to figure out why dependabot made it though, as I would have expected `^5.0.0` (and for many packages moving to the next major version is what we'll want for generic dep changes - we don't test the low end of ranges and the high end for most packages).

Reply:
As a matter of fact I would strictly prefer to widen. I think it would only harm the version solving if we strictly bump.

Reply:
I stand corrected:
https://github.com/dependabot/dependabot-core/blob/a2143f63bd25804ec9aec65b9ebb4d38487b1ebd/pub/lib/dependabot/pub/update_checker.rb#L93-L116

Filed https://github.com/dependabot/dependabot-core/issues/4979 for further discussion.